packages/access-api/test/provider-add.test.js

```diff
@@ -97,6 +99,95 @@ for (const accessApiVariant of /** @type {const} */ ([
       })
     })
   })
+
+  if (
+    ['provider/add', 'access/delegate'].every((f) =>
+      accessApiVariant.features.has(f)
+    )
+  ) {
+    it('provider/add allows for access/delegate', async () => {
+      const space = await principal.ed25519.generate()
+      const agent = await accessApiVariant.issuer
+      const service = await accessApiVariant.audience
+      const accountDid = /** @type {const} */ ('did:mailto:example.com:foo')
+
+      const accountAuthorizesAgentClaim = await ucanto.delegate({
+        issuer: principal.Absentee.from({ id: accountDid }),
+        audience: agent,
+        capabilities: [
+          {
+            with: 'ucan:*',
+            can: '*',
+          },
+        ],
+      })
+      const serviceAttestsThatAccountAuthorizesAgent = await ucanto.delegate({
+        issuer: service,
+        audience: agent,
+        capabilities: [
+          {
+            with: service.did(),
+            can: 'ucan/attest',
+            nb: { proof: accountAuthorizesAgentClaim.cid },
+          },
+        ],
+      })
+      const sessionProofs = [
+        accountAuthorizesAgentClaim,
+        serviceAttestsThatAccountAuthorizesAgent,
+      ]
+      const addStorageProvider = await ucanto
+        .invoke({
+          issuer: agent,
+          audience: service,
+          capability: {
+            can: 'provider/add',
+            with: accountDid,
+            nb: {
+              provider: 'did:web:web3.storage:providers:w3up-alpha',
+              consumer: space.did(),
+            },
+          },
+          proofs: [...sessionProofs],
+        })
+        .delegate()
+      const addStorageProviderResult = await accessApiVariant.invoke(
+        addStorageProvider
+      )
+      assertNotError(addStorageProviderResult)
+
+      // storage provider added. So we should be able to delegate now
+      const accessDelegate = await ucanto
+        .invoke({
+          issuer: agent,
+          audience: service,
+          capability: {
+            can: 'access/delegate',
+            with: space.did(),
+            nb: {
+              delegations: {},
+            },
+          },
+          proofs: [
+            ...sessionProofs,
+            // space says agent can access/delegate with space
+            await ucanto.delegate({
+              issuer: space,
+              audience: agent,
+              capabilities: [
+                {
+                  can: 'access/delegate',
+                  with: space.did(),
+                },
+              ],
+            }),
```

Comment:
This isn't used by anyhing as far as I can tell. And `sessionProofs` should be enough to call `access/delegate` with `space.did()`, so motivation here is unclear.

Reply:
how would session proofs be enough to act on `with=space.did()`?

Reply:
without this proof the test fails because agent is now allowed to `access/delegate` with space

Reply:
we spoke in person and as a result I realized that the `sessionProofs` weren't needed since we have that one, and we removed them https://github.com/web3-storage/w3protocol/pull/483/commits/940d24cd9f2626e69fc2e2210559c4ad9516576d